### PR TITLE
Added product Description feature file for AZ-53 story

### PR DESCRIPTION
**Pull request by abhilashawagh**
Hello Abhishek,

Please review the feature file for story AZ-53 and let me know the changes if any.

Thanks, 
Abhilasha Wagh.

### CODE REVIEW

File: src/test/resources/features/prodDescription.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@ProductDesc
+Feature: Amazon health check
+  
+  @tag1
+  Scenario: Product description for amazon  %$45
```
Comment (Abhishek8694) [inline src/test/resources/features/prodDescription.feature]: Please check the scenario line. Its having "%$45". Please remove it.

File: src/test/resources/features/prodDescription.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+
+  @tag2 -----@123
```
Comment (Abhishek8694) [inline src/test/resources/features/prodDescription.feature]: Please check the tagline. Its having "-----@123". Please remove it.

**Review by Abhishek8694 — COMMENTED** [review]
Please make the required changes in feature file for **AZ-53** story.
Have a Nice Day

File: src/test/resources/features/prodDescription.feature (hunk 1 above)
Comment (abhilashawagh) [reply src/test/resources/features/prodDescription.feature]: I made changes at scenario line

File: src/test/resources/features/prodDescription.feature (hunk 2 above)
Comment (abhilashawagh) [reply src/test/resources/features/prodDescription.feature]: I made changes at tagline.

**Comment by Abhishek8694** [comment]
OK, now you can merge the branch into the master.

**Pull request merged**